### PR TITLE
[FIX] mail: do not show chat window hidden menu in mobile

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2795429

### CODE REVIEW

File: addons/mail/static/src/components/chat_window_manager/chat_window_manager.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <div class="o_ChatWindowManager" t-attf-class="{{ className }}" t-ref="root">
             <t t-if="messaging and messaging.chatWindowManager">
                 <!-- Note: DOM elements are ordered from left to right -->
-                <t t-if="messaging.chatWindowManager.hasHiddenChatWindows">
+                <t t-if="messaging.chatWindowManager.hasHiddenChatWindows and messaging.chatWindowManager.visual.hidden.isVisible">
```
Comment (seb-odoo) [inline addons/mail/static/src/components/chat_window_manager/chat_window_manager.xml:8]: Remove field `hasHiddenChatWindows`?

For later: we need to use models (or probably chat window itself) to kill this visual thing.

Comment (alexkuhn) [reply addons/mail/static/src/components/chat_window_manager/chat_window_manager.xml]: Done

**Comment by alexkuhn** [comment]
@seb-odoo just added a test

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**